session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -1179,35 +1179,14 @@ public void insertRecordsOfOneDevice(
       List<List<TSDataType>> typesList,
       List<List<Object>> valuesList)
       throws IoTDBConnectionException, StatementExecutionException {
```

Comment:
two overload function which only differs in `haveSorted param`, thus merged to one